fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/ColumnParser.java

```diff
@@ -186,4 +192,61 @@ public boolean parse(String value) {
             throw new RuntimeException("string check failed ", e);
         }
     }
+}
+
+class DecimalParser extends ColumnParser {
+
+    public static int PRECISION = 27;
+    public static int SCALE = 9;
+
+    private BigDecimal maxValue;
+    private BigDecimal minValue;
+
+    public DecimalParser(EtlJobConfig.EtlColumn etlColumn) {
+        StringBuilder precisionStr = new StringBuilder();
+        for (int i = 0; i < etlColumn.precision; i++) {
+            precisionStr.append("9");
+        }
+        StringBuilder scaleStr = new StringBuilder();
+        for (int i = 0; i < etlColumn.scale; i++) {
+            scaleStr.append("9");
+        }
+        maxValue = new BigDecimal(precisionStr.toString() + "." + scaleStr.toString());
+        minValue = new BigDecimal("-" + precisionStr.toString() + "." + scaleStr.toString());
+    }
+
+    @Override
+    public boolean parse(String value) {
+        try {
+            BigDecimal bigDecimal = new BigDecimal(value);
+            return bigDecimal.precision() - bigDecimal.scale() <= PRECISION - SCALE && bigDecimal.scale() <= SCALE;
+        } catch (NumberFormatException e) {
+            return false;
+        } catch (Exception e) {
+            throw new RuntimeException("decimal parse failed ", e);
+        }
+    }
+
+    public BigDecimal getMaxValue() {
+        return maxValue;
+    }
+
+    public BigDecimal getMinValue() {
+        return minValue;
+    }
+}
+
+class LargeIntParser extends ColumnParser {
+
+    @Override
+    public boolean parse(String value) {
+        try {
+            BigInteger bigInteger = new BigInteger(value);
```

Comment:
Check largeint type range

Reply:
>BigInteger constructors and operations throw {@code ArithmeticException} when
 >the result is out of the supported range of
 > -2 Integer.MAX_VALUE}(exclusive) to
 > +2Integer.MAX_VALUE}(exclusive)
So I think add a ArithmeticException here is enough